src/main/scala/dotty/dokka/tasty/BasicSupport.scala

```diff
@@ -115,8 +127,11 @@ class SymOps[R <: Reflection](val r: R):
         Flags.Lazy -> Modifier.Lazy,
         Flags.Override -> Modifier.Override,
         Flags.Case -> Modifier.Case,
-        ).collect { case (flag, mod) if sym.flags.is(flag) => mod }
+        ).collect { 
```

Comment:
why this change? Was it an autoformat?

Reply:
Idk, probably I was debugging sth and rewrote it this way, refactored